src/blocks/ImageTextBlock.tsx

```diff
@@ -137,14 +155,17 @@ interface ImageTextBlockProps extends BaseBlockProps {
   size: SectionSize
   media_position: 'top' | 'bottom'
   button_color?: ColorComponent
+  button_size?: keyof typeof buttonSizes
   button_weight?: ButtonWeight
+  button_position_mobile?: 'above' | 'below'
 }
 
 export const ImageTextBlock: React.FunctionComponent<ImageTextBlockProps> = ({
   title_size,
   title,
   paragraph,
   text_position,
+  text_alignment_mobile,
```

Comment:
is there a difference between `alignment` and `position`? Otherwise i think we should pick one of them

Reply:
Makes sense, changing to text_position_mobile